pkg/clusterapi/system.go

```diff
@@ -150,7 +150,7 @@ func (c *system) Run(ctx context.Context) error {
 				"--health-addr={{suggestHealthHostPort}}",
 				"--webhook-port={{.WebhookPort}}",
 				"--webhook-cert-dir={{.WebhookCertDir}}",
-				"--feature-gates=BootstrapFormatIgnition=true,ExternalResourceGC=true,TagUnmanagedNetworkResources=false",
+				"--feature-gates=BootstrapFormatIgnition=true,ExternalResourceGC=true,TagUnmanagedNetworkResources=false,EKS=false",
```

Comment:
@r4f4 looks like FG `AutoControllerIdentityCreator`([1][1]) is also enabled([2][2]) by default in the current version, are we using it or should we disable it?

I was looking if we have an easy way to "disable-all" then enable desired to prevent upstream changes introducing new unwanted features, or disable something we are using, but it looks like there is not possible.

[1]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/888c6596e3a1bdfde0f8235011cca56a5cc117c1/feature/feature.go#L62-L65
[2]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/888c6596e3a1bdfde0f8235011cca56a5cc117c1/feature/feature.go#L105

Reply:
@mtulio we use that one [here](https://github.com/openshift/installer/blob/master/pkg/asset/manifests/aws/cluster.go#L260-L270)

Reply:
But one that is currently disabled by default and we could add to the list is the [ROSA one](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/888c6596e3a1bdfde0f8235011cca56a5cc117c1/feature/feature.go#L88)